lib/auth-middleware/setup.py

```diff
@@ -27,9 +27,11 @@
     description='Rokwire Platform Auth Middleware',
     # long_description=open('README.txt').read(),
     install_requires=[
-        'PyJWT>=1.7.1,<2',
-        'requests>=2.22.0,<3',
-        'cryptography>=3,<4',
+        'PyJWT>=1.7.1',
+        'requests>=2.22.0',
+        'cryptography>=2.7',
+        'requests_oauthlib>=1.3.0'
```

Comment:
This one can be gone. Probably, we don't need any auth-middleware change at all in here.

Reply:
ok, I see, then i can probably remove them